seqr/views/apis/anvil_workspace_api.py

```diff
@@ -110,6 +110,13 @@ def create_project_from_workspace(request, namespace, name):
     if errors:
         return create_json_response({'errors': errors}, status=400)
 
+    # Validate the VCF to see if it contains all the required samples
+    samples = get_vcf_samples(data_path)
```

Comment:
`get_vcf_samples` returns empty in the scenario where we are not able to get a properly formatted header row and parse the sample IDs, and in that case we need to return an error that explains that instead of just giving an error that says all the samples are missing

Reply:
Changed to respond with an error if the `samples` is empty.